src/components/TransactionHistoryRows/TransactionGenericRow.tsx

```diff
@@ -39,7 +39,6 @@ export const GetTxLayoutFormats = ({ parentWidth }: { parentWidth: number }) =>
 
   if (isLargerThanSm) {
     columns = '1fr 2fr'
-    dateFormat = 'LT'
   }
   if (isLargerThanMd) {
     columns = '1fr 2fr'
```

Comment:
Do we want to just remove this guy now that it's effectively the same as the sm breakpoint above?

Reply:
yah i'll move it out as a const